docs/contributing/writing-instrumentation-module.md

```diff
@@ -360,3 +360,203 @@ For example:
 ```
 
 [suppress]: https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#suppressing-specific-auto-instrumentation
+
+## Use non-inlined advice code with `invokedynamic`
+
+Using non-inlined advice code is possible thanks to the `invokedynamic` instruction, this strategy
+is referred as "indy" in reference to this. By extension "indy modules" are the instrumentation
+modules using this instrumentation strategy.
+
+The most common way to instrument code with bytebuddy relies on inlining, this strategy will be
```

Comment:
Suggest to use `ByteBuddy` instead of `bytebuddy` here.

Reply:
Fixed in ac90f56935fb78ef005606756739e7e871d2aa7e